utils/src/main/kotlin/Utils.kt

```diff
@@ -62,9 +62,10 @@ inline fun <reified T : Any> T.hasNonNullProperty() =
     T::class.memberProperties.asSequence().map { it.get(this) }.any { it != null }
 
 /**
- * Filter a list of [names] to include only those that likely belong to the given [version] of an optional [project].
+ * Filter a list of [names] to include only those that likely belong to the given [version] of an optional
+ * [packageName].
  */
-fun filterVersionNames(version: String, names: List<String>, project: String? = null): List<String> {
+fun filterVersionNames(version: String, names: List<String>, packageName: String? = null): List<String> {
```

Comment:
If we change this, I wonder whether we shouldn't use something more neutral here, like `prefix`, as at least in the current implementation `packageName` is only supported as a prefix.